fission-core/library/Fission/User/DID/Types.hs

```diff
@@ -113,13 +116,11 @@ instance Display DID where -- NOTE `pk` here is base2, not base58
       multicodecW8 :: [Word8]
       multicodecW8 =
         case pk of
-          Ed25519PublicKey ed  -> 0xed : 0x01 : BS.unpack (BA.convert ed)
-          RSAPublicKey     rsa -> 0x00 : 0xF5 : 0x02 : BS.unpack (BS64.decodeLenient . encodeUtf8 $ textDisplay rsa)
-                               {-   ^     ^     ^
-                                    |     |     |
-                                    |    "expect 373 Bytes", encoded in the mixed-endian format
-                                  "raw"
-                              -}
+          Ed25519PublicKey ed ->
+            0xed : 0x01 : BS.unpack (BA.convert ed)
+
+          RSAPublicKey rsa ->
+            0x12 : 0x05 : BS.unpack (Lazy.toStrict . Builder.toLazyByteString . getUtf8Builder $ display rsa)
```

Comment:
:raised_hands:

Reply:
Just randomly took a stroll through here, and now I know this is actually incorrect, the prefix needs to be `0x85 : 0x24 : Bs.unpack ...`
I've implemented it that way in ts-ucan and it tests fine with the test vectors from the DID spec.
https://github.com/ucan-wg/ts-ucan/pull/42/files#diff-109e954c94a80ae8fd78b2ce9734642de7a787f48926c9a5ba18ad1ec76225ffR17

The reason it's `0x85 0x24` is that that's the number `0x1205` encoded as unsigned varint.
Just like a couple of lines above, the multicodec for ed25519-pub `0xed` is encoded as unsigned varint into `0xed 0x01`.